src/promptflow-evals/promptflow/evals/synthetic/_model_tools/_proxy_completion_model.py

```diff
@@ -99,6 +99,7 @@ async def request_api(
         proxy_headers = {
             "Authorization": f"Bearer {token}",
             "Content-Type": "application/json",
+            "User-Agent": "promptflow-evals",

```

Comment:
Lets pick this value from user agent file ? That gives both package name and version of pf evals package

Reply:
Changed!